docs/GETTING_STARTED.md

```diff
@@ -26,3 +26,47 @@ If you compile dlib with CUDA make sure you add the appropriate includes / flags
 You can download MKL [here](https://software.intel.com/en-us/mkl). Follow the instructions for installation, then run `scripts/bootstrap.sh` and recompile dlib with MKL support.
 
 Then make sure that the appropriate includes / flags in the `addons_config.make` file are available.
+
+### On Visual Studio 
+
+Since there is no script yet, you need to download and compile dlib by yourself. 
+
+#### Download dlib
+
+[http://dlib.net/]
+
+#### Compile
+You need CMake installed [https://cmake.org/]. Make sure you put the CMake folder in your Path system variable (default is `c:\Program Files\CMake\bin`).
+
+Then configure and compile dlib using these commands in the terminal (opfen the dlib folder in the terminal)
+```
+mkdir build
+cd build
+
+cmake -G "Visual Studio 15 2017" -A x64 -DDLIB_USE_CUDA=1 -DUSE_AVX_INSTRUCTIONS=1 -DJEPG_INCLUDE_DIR=..\dlib\external\libjpeg -DJEPG_LIBRARY=..\dlib\external\libjpeg -DPNG_PNG_INCLUDE_DIR=..\dlib\external\libpng -DPNG_LIBRARY_RELEASE=..\dlib\external\libpng -DZLIB_INCLUDE_DIR=..\dlib\external\zlib \-DZLIB_LIBRARY_RELEASE=..\dlib\external\zlib -DCMAKE_INSTALL_PREFIX=install ..```
+
+```
+cmake --build . --config Debug --target INSTALL
+cmake --build . --config Release --target INSTALL
+```
+
+(This instruction roughly followed [https://medium.com/beesightsoft/build-dlib-on-windows-534209e8340a])
+
+#### Prepare ofxDlib
+Now copy the dlib includes and libraries into the addon folder.
+
+* folder `dlib` from `[dlib download folder]` to `[of addons folder]/ofxDlib/libs/dlib/include`
```

Comment:
Cmake generates custom header files under `/build/install` folder.
So we should copy `/build/install/dlib` folder.
Otherwise `#define DLIB_USE_CUDA`  is commented out in `[dlib download folder]/dlib/config.h `